### ISSUE TITLE
add one section in README introducing deposed-based tools.

### ISSUE THREAD

**Issue by promeG** [alibaba/dexposed#25]
Thanks for creating the awesome dexposed framework!

Recently I have developed [XLog module](https://github.com/promeG/XLog) based on dexposed. XLog can easily log method calls and print arguments, return values, and the execute time. After using XLog in our own project for a while, we found that it is very useful in debugging and performance monitoring and want to share this tool with more developers.

I wonder if it is appropriate to add one section in README introducing dexposed-based tools such as Xlog, so that the benefits brought by dexposed framework could be enjoyed by more developers.

**Comment by pcqpcq** [comment]
nice idea! 
I think maybe put it in wiki is another good way, beyond the readme file.

**Comment by hwjump** [comment]
Yes, I agree with pcqpcq.  I will put some dexposed-base tools in another file!  
Thank you for your work!

**Comment by UserWang** [comment]
and then？
